### DISCUSSION TITLE
Welcome to #RSEAA24 Posters!!!

### DISCUSSION THREAD

**Discussion by richarddushime** [RSEAA/RSEAA.github.io#75] [Announcements]
## 👋 Welcome!
  We’re using GitHub Discussions as a place for #RSEAA24 posters as well as to guide the discussion topics of our first, second and third sessions of the unconference. 

## Posters
We hope that you would like to contribute by creating a poster by:
1.  uploading an accessible powerpoint document/image / pdf to [Zenodo](https://zenodo.org/communities/rseaa/) and adding the keyword #RSEAA24 - you can follow the [instructions to upload to Zenodo here](https://rb.gy/7cl10g).
2. creating a discussion under the category "Posters for #RSEAA24" on this board and putting the link to the Zenodo Presentation

We will create an example of a poster for #RSEAA24 soon.

The idea of the poster would be to possibly:
  * introduce yourself 
  * share where you work 
  * ask a question of the community or a talking point - "What tool are people using for X?"
  * suggest a solution  - "We have had this problem X and we solved it with Y"

... based on whatever you are comfortable with.

### Look through the current Posters

You can see all the poster topics by:
1. [Latest activity ](https://github.com/RSEAA/RSEAA.github.io/discussions/categories/posters-for-rseaa24?discussions_q=is%3Aopen+category%3A%22Posters+for+%23RSEAA24%22) or
2. [Most popular](https://github.com/RSEAA/RSEAA.github.io/discussions/categories/posters-for-rseaa24?discussions_q=is%3Aopen+category%3A%22Posters+for+%23RSEAA24%22+sort%3Atop) or
3. [View directly in Zenodo](https://zenodo.org/communities/rseaa/)

## Discussion Topics

We also [have Discussion Topics that will guide our discussions at the unconference](https://github.com/RSEAA/RSEAA.github.io/discussions/43).

**Comment by GenevieveBuckley** [comment]
fyi @richarddushime - the link you have above to "view directly in figshare" resolves to a URL containing the keyword "RSEAA2023", not "RSEAA2024". I see two documents at the 2023 link, and nothing under the 2024 keyword.

**Reply by rowlandm** [reply]
Thanks for finding that @GenevieveBuckley ! I changed it to Zenodo.
